frontend/app/src/components/history/transactions/TransactionEventNote.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <template v-for="(note, index) in formatNotes(notes, amount, asset)">
+      <span
+        v-if="note.type === 'address' || note.type === 'tx'"
+        :key="index"
+        class="d-inline-flex"
+      >
+        <hash-link
+          :class="$style['address']"
+          :text="note.address"
+          :tx="note.type === 'tx'"
+        />
+      </span>
+      <span v-else-if="note.type === 'amount'" :key="index">
+        <amount-display :asset="note.asset" :value="note.amount" />
+      </span>
+      <span v-else :key="index">
+        {{ note.word }}
+      </span>
+    </template>
+  </div>
+</template>
+<script lang="ts">
+import { BigNumber } from '@rotki/common';
+import { defineComponent } from '@vue/composition-api';
+import { get } from '@vueuse/core';
+import { useAssetInfoRetrieval } from '@/store/assets';
+import { bigNumberify } from '@/utils/bignumbers';
+
+export default defineComponent({
+  name: 'TransactionEventNote',
+  props: {
+    notes: { required: false, type: String, default: '' },
+    amount: { required: false, type: BigNumber, default: null },
```

Comment:
Does this work fine? I seem to recall that if we don't do type: Object as PropType<BigNumber> it usually complains somewhere.

Reply:
Now it dont complains anymore. I actually figured it out when do vite integration, using `type: Object as PropType<BigNumber>` give me a lot of warns. Then I tried to write it this way and it stop give me warns.